a10_octavia/common/config_options.py

```diff
@@ -85,6 +85,9 @@
 A10_HEALTH_MONITOR_OPTS = [
     cfg.StrOpt('post_data',
                help=_('HTTP Content for "--http-method POST" case.')),
+    cfg.BoolOpt('use_override_port',
```

Comment:
We don't want this as an option nor do we want it to be true by default. We want the override port change here https://github.com/a10networks/acos-client/blob/master/acos_client/v30/slb/hm.py#L96

Reply:
Fixed